assets/src/parse_bundle_plugin.js

```diff
@@ -16,9 +16,9 @@ var _ = require('lodash');
 /**
  * Turn an object containing the vital information for a frontend plugin and return a bundle configuration for webpack.
  * @param {Object} data - An object that contains the data for configuring the bundle.
- * @param {string} data.entry_file - The Javascript source file that initializes the plugin.
```

Comment:
I used 'entry_file' here, as that is Webpack idiom - also, because it is the entry point into the bundle (a single file does not define all the source code, but it is the root node in the source tree for the bundle).